tables/dmadata_table.txt

```diff
@@ -31,7 +31,7 @@
     ('build/baserom/assets/textures/nes_font_static', '', 0x1000, 0),
     ('build/baserom/assets/textures/message_data_static', '', 0x1000, 0),
     ('build/baserom/assets/textures/staff_message_data_static', '', 0x1000, 0),
-    ('build/baserom/code', 'build/comp/code.yaz0', 0x1000, 0),
+    ('build/binary/code', 'build/comp/code.yaz0', 0x1000, 0),
```

Comment:
This is actually a bug like #121 - where the recompiled `code` section wasn't being incorporated into the ROM.